packaging/windows/OpenRA.nsi

```diff
@@ -18,6 +18,7 @@
 !include "MUI2.nsh"
 !include "FileFunc.nsh"
 !include "WordFunc.nsh"
+!include "nsProcess.nsh"

```

Comment:
That should work according to http://help.appveyor.com/discussions/questions/303-how-to-unzip-a-file-in-the-install-script :white_check_mark: We can only test this when tagging a new release.